ansible/roles/texlive/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+texlive_dest: /opt/texlive
+texlive_version: 2015
+# The above is duplicated (!) in the installer profile file - ../files/
```

Comment:
Is there a need to customise the vars below independent of the above? If not keep it simple and put them straight into a template.